src/app/auth/shared/auth-form/auth-form.component.html

```diff
@@ -0,0 +1,64 @@
+<form [formGroup]="form" (ngSubmit)="onSubmit()" class="form-container">
```

Comment:
This could just be moved to the auth component, as common for both pages, and the form itself would be loaded depending on the selected page. You should have this entire form wrapped in <mat-card />